migen/build/platforms/sinara/kasli_soc.py

```diff
@@ -0,0 +1,314 @@
+from migen.build.generic_platform import *
+from migen.build.xilinx import XilinxPlatform
+
+
+_io = [
+    ("clk500", 0, Pins("B24"), IOStandard("LVCMOS25")),
```

Comment:
That's a PS pin so shouldn't be here, and also that clock is 33MHz.

Reply:
The pin is named `PS_CLK_500`. Removing...